### PR TITLE
New package: musl-locales-0.git20201006

### PR DESCRIPTION
**Pull request by Logarithmus**
Partial locale support for `musl`
Homepage: https://gitlab.com/rilian-la-te/musl-locales

### CODE REVIEW

**Comment by ericonr** [comment]
Do you know if there are plans for a release? Would be nice to have.

**Comment by fosslinux** [comment]
> Please note that we do not accept any packages containing non-release versions, such as specific git- or svn-revisions anymore.

https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md

File: srcpkgs/musl-locales/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+pkgname=musl-locales
+version=0.git20201006
+revision=1
+_commit=1101fb2bcdd189cd9415b8bd1c775eb43527d25c
+archs="*-musl"
+wrksrc=${pkgname}-${_commit}
+build_style=cmake
+hostmakedepends="gettext"
+makedepends="musl-devel gettext-stub-devel gettext-devel"
+short_desc="Locales support for musl"
```
Comment (ericonr) [inline srcpkgs/musl-locales/template:11]: Use upstream desc: "Locale program for musl libc"

**Review by ericonr — COMMENTED** [review]
I am quite interested in this project, but hadn't gotten around to making a template.

Feel free to poke upstream on my behalf for a release, I would really like to get this into void.

File: srcpkgs/musl-locales/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'musl-locales'
+pkgname=musl-locales
+version=0.git20201006
+revision=1
+_commit=1101fb2bcdd189cd9415b8bd1c775eb43527d25c
+archs="*-musl"
+wrksrc=${pkgname}-${_commit}
+build_style=cmake
+hostmakedepends="gettext"
+makedepends="musl-devel gettext-stub-devel gettext-devel"
```
Comment (ericonr) [inline srcpkgs/musl-locales/template:10]: `musl-devel` is available by default

Comment (ericonr) [reply srcpkgs/musl-locales/template:10]: why is stub also required?

Comment (ericonr) [reply srcpkgs/musl-locales/template:10]: It builds fine with

```diff
diff --git a/srcpkgs/musl-locales/template b/srcpkgs/musl-locales/template
index 704b89c7e7..e8933b9722 100644
--- a/srcpkgs/musl-locales/template
+++ b/srcpkgs/musl-locales/template
@@ -7,7 +7,7 @@ archs="*-musl"
 wrksrc=${pkgname}-${_commit}
 build_style=cmake
 hostmakedepends="gettext"
-makedepends="musl-devel gettext-stub-devel gettext-devel"
+makedepends="gettext-devel"
 short_desc="Locales support for musl"
 maintainer="Artur Sinila <opensource@logarithmus.dev>"
 license="LGPL-3.0-only, MIT"
```

File: srcpkgs/musl-locales/template (hunk 1 above)
Comment (ericonr) [reply srcpkgs/musl-locales/template]: Ignore this, the short desc is good.

**Comment by ndowens** [comment]
Yep Void requires upstream to have a release instead of git tags.  Create issue upstream asking for a release so we can have in repo and link it here

**Comment by Logarithmus** [comment]
Maybe the author will notice it sometime...
https://gitlab.com/rilian-la-te/musl-locales/-/issues/11

**Comment by ericonr** [comment]
Given that upstream hasn't responded to the issue yet, I will be closing this for now. If we are very lucky, musl might come up with their own solution.

Feel free to reopen once the situation changes.

**Pull request closed without merging**